vendor/wp-now/src/wp-now.ts

```diff
@@ -602,6 +602,20 @@ async function mountInternalMuPlugins( php: PHP ) {
 		});
 		`
 	);
+
+	// Only add the auto-updates disabling plugin for non-latest WordPress versions
+	if ( options?.wordPressVersion && options.wordPressVersion !== 'latest' ) {
+		php.writeFile(
+			path.posix.join( PLAYGROUND_INTERNAL_MU_PLUGINS_FOLDER, '0-disable-auto-updates.php' ),
+			`<?php
+			// Disable auto-updates
+			add_filter( 'auto_update_core', '__return_false' );
```

Comment:
I guess this is not really necessary with the other three filters (dev, major and minor).
In fact, [`auto_udpate_core`](https://developer.wordpress.org/reference/hooks/auto_update_type/#description) is not working for me at all. I tried using only this hook in beta and old stable versions and is not making any effect. I think we can remove it.